src/libraries/Microsoft.Extensions.DependencyInjection/src/ServiceLookup/ILEmit/ILEmitResolverBuilder.cs

```diff
@@ -485,5 +497,17 @@ private void Stloc(ILGenerator generator, int index)
 
             generator.Emit(OpCodes.Stloc, index);
         }
+
```

Comment:
With my suggestions above, this function is not needed. It would be great if you accordingly remove the need for the `Ldloc` and `Stloc` helper methods above.